src/uct/ib/dc/dc_mlx5_ep.h

```diff
@@ -308,12 +308,12 @@ uct_dc_mlx5_ep_basic_init(uct_dc_mlx5_iface_t *iface, uct_dc_mlx5_ep_t *ep)
 
     if (uct_dc_mlx5_iface_is_dci_rand(iface)) {
         /* coverity[dont_call] */
-        ep->dci                   = rand_r(&iface->tx.rand_seed) % iface->tx.ndci;
-        ep->dci_stream_channel_id = iface->tx.dcis[ep->dci].dci_channel_index++ %
-                                    UCS_BIT(iface->tx.log_num_dci_stream_channels);
+        ep->dci               = rand_r(&iface->tx.rand_seed) % iface->tx.ndci;
+        ep->dci_channel_index = iface->tx.dcis[ep->dci].next_channel_index++ &
+                                UCS_MASK(iface->tx.num_dci_channels);
```

Comment:
can we precompute `UCS_MASK(iface->tx.num_dci_channels)` and save on the iface? instead of saving tx.num_dci_channels

Reply:
> can we precompute `UCS_MASK(iface->tx.num_dci_channels)` and save on the iface? instead of saving tx.num_dci_channels

this code is not performance critical, do you think storing pre-calculated value is reasonable here?
not actual anymore: used non-log value and `%` operation